src/typelib.erl

```diff
@@ -91,8 +91,8 @@ remove_pos({Type, _, Value})
 remove_pos({user_type, Anno, Name, Params}) when is_list(Params) ->
     {user_type, anno_keep_only_filename(Anno), Name,
      lists:map(fun remove_pos/1, Params)};
-remove_pos({type, Anno, record, Params = [_Name]}) ->
-    {type, anno_keep_only_filename(Anno), record, Params};
+remove_pos({type, Anno, record, Params = [{atom, AtomAnno, Name}]}) ->
+    {type, anno_keep_only_filename(Anno), record, [{atom, anno_keep_only_filename(AtomAnno), Name}]};
```

Comment:
Omg we were not fixing the anno for the record's atom name @_@ lol.

Reply:
Good. :-)